Ginger/Ginger/Reports/ValidationRules/ValidateNotContainSpecificChar.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Controls;
+
+namespace Ginger.Reports.ValidationRules
+{
+    public class ValidateNotContainSpecificChar : ValidationRule
+    {
+        private string _Message = string.Empty;
+
+        public ValidateNotContainSpecificChar(string message = "Variable name can't contain a comma (',')")
```

Comment:
i didn't got why to set the error message in the constructor and not to get in it the char to validate?
Validation rule is called 'ValidateNotContainSpecificChar' so it needs to be generic and validate any char and not only comma